packages/lesswrong/components/sunshineDashboard/SunshineNewUsersInfo.tsx

```diff
@@ -198,9 +200,9 @@ const SunshineNewUsersInfo = ({ user, classes, refetch, currentUser }: {
                 <div>
                   <UserReviewStatus user={user}/>
               
-                  {user.associatedClientId?.firstSeenReferrer && <div className={classes.qualitySignalRow}>Initial referrer: {user.associatedClientId?.firstSeenReferrer}</div>}
-                  {user.associatedClientId?.firstSeenLandingPage && <div className={classes.qualitySignalRow}>Initial landing page: {user.associatedClientId?.firstSeenLandingPage}</div>}
-                  {(user.associatedClientId?.userIds?.length??0) > 1 && <div className={classes.qualitySignalRow}>
+                  {firstClientId?.firstSeenReferrer && <div className={classes.qualitySignalRow}>Initial referrer: {firstClientId?.firstSeenReferrer}</div>}
```

Comment:
Make sure to migrate this into UserReviewStatus

Reply:
This one showed up as a merge conflict, so was guaranteed to not be missed